index.bs

```diff
@@ -431,11 +431,361 @@ A file containing a 'pict' track compliant with this profile is expected to list
+        <td>&nbsp;</td>
+        <td>Used when an image is auxiliary to another image</td>
+    </tr>
+    <tr>
+        <td>&nbsp;</td>
+        <td>thmb</td>
+        <td>-</td>
+        <td>HEIF</td>
+        <td>&nbsp;</td>
+        <td>Used when an image is a thumbnail of another image</td>
+    </tr>
+    <tr>
+        <td>&nbsp;</td>
+        <td>dimg</td>
+        <td>-</td>
+        <td>HEIF</td>
+        <td>&nbsp;</td>
+        <td>Used when an image is derived from another image</td>
+    </tr>
+    <tr>
+        <td>&nbsp;</td>
+        <td>prem</td>
+        <td>-</td>
+        <td>MIAF</td>
+        <td>&nbsp;</td>
+        <td>Used when when an alpha image contains premultiplied color values from another image</td>
+    </tr>
+    <tr>
+        <td>&nbsp;</td>
+        <td>cdsc</td>
+        <td>-</td>
+        <td>HEIF</td>
+        <td>&nbsp;</td>
+        <td>Used to link metadata with an image</td>
+    </tr>
+    <tr>
+        <td>idat</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>ISOBMFF</td>
+        <td>meta</td>
+        <td>Used to store derived image definitions</td>
+    </tr>
+    <tr>
+        <td>grpl</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>ISOBMFF</td>
+        <td>meta</td>
+        <td>Used to indicate that multiple images are semantically grouped</td>
+    </tr>
+    <tr>
+        <td>&nbsp;</td>
+        <td>altr</td>
+        <td>0</td>
+        <td>ISOBMFF</td>
+        <td>&nbsp;</td>
+        <td>Used when images in a group are alternative to each other</td>
+    </tr>        
+    <tr>
+        <td>pasp</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>ISOBMFF</td>
+        <td>ipco</td>
+        <td>Used to signal pixel aspect ratio. If present, shall indicate a pixel aspect ratio of 1:1</td>
+    </tr>
+    <tr>
+        <td>colr</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>ISOBMFF</td>
+        <td>ipco</td>
+        <td>Used to signal information such as color primaries.</td>
+    </tr>
+    <tr>
+        <td>auxC</td>
+        <td>&nbsp;</td>
+        <td>0</td>
+        <td>HEIF</td>
+        <td>ipco</td>
+        <td>Used to signal the type of an auxiliary image (e.g. alpha, depth).</td>
+    </tr>
+    <tr>
+        <td>clap</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>ISOBMFF</td>
+        <td>ipco</td>
+        <td>Used to signal cropping applied to an image</td>
+    </tr>
+    <tr>
+        <td>irot</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>HEIF</td>
+        <td>ipco</td>
+        <td>Used to signal a rotation applied to an image</td>
+    </tr>
+    <tr>
+        <td>imir</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>HEIF</td>
+        <td>ipco</td>
+        <td>Used to signal a mirroring applied to an image</td>
+    </tr>
+    <tr>
+        <td>clli</td>
+        <td>&nbsp;</td>
+        <td>-</td>
+        <td>ISOBMFF</td>
+        <td>ipco</td>
```

Comment:
Note that `clli` is a `Box` and not an `ItemProperty`.  
ISOBMFF says
```
aligned(8) class ItemPropertyContainerBox extends Box('ipco')
{
   Box properties[]; // boxes derived from
   // ItemProperty or ItemFullProperty, or FreeSpaceBox(es)
   // to fill the box
}
```
fortunately ISOBMFF also says
> Each item property is a Box or FullBox.

so I think this is just another inconsistency of ISOBMFF and there is no issue here.